website/src/docs/hotchocolate/integrations/neo4j.md

```diff
@@ -15,6 +15,17 @@ To use the Neo4J integration, you need to install the package `HotChocolate.Data
 dotnet add package HotChocolate.Data.Neo4J
 ```
 
+To use the HotChocolate AspNetCore integration, you need to install the package `HotChocolate.AspNetCore`.
+
+```bash
+dotnet add Neo4j package HotChocolate.AspNetCore
+```
+
+```csharp
+services
```

Comment:
This seems incomplete

Reply:
Hi @michaelstaib, the intent of this code snippet was to show why is it necessary to also add the  `HotChocolate.AspNetCore` package. Should I include something else here, too? I thought that a basic simple example will be the best in this place. But let me know if we can improve it of course.

Thank you very much for the review!

Reply:
I think this hold section about AspNetCore is a bit confusing. Its also not necessary to have this package.
Lets kick this out and then we can merge the fixes that you have done to the docs.